website/index.html

```diff
@@ -41,19 +44,301 @@ <h2>OSC-bridge for controlling ATEM switchers</h2>
+              <li>
+                Black <span class="callout">/atem/preview/0</span>
+              </li>
+              <li>
+                Input 1 <span class="callout">/atem/preview/1</span>
+              </li>
+              <li>
+                Input 2 <span class="callout">/atem/preview/2</span>
+              </li>
+              <li>
+                Input 3 <span class="callout">/atem/preview/3</span>
+              </li>
+              <li>
+                Input 4 <span class="callout">/atem/preview/4</span>
+              </li>
+              <li>
+                and so on...
+              </li>
+              <li>
+                Bars <span class="callout">/atem/preview/7</span>
+              </li>
+              <li>
+                Color 1 <span class="callout">/atem/preview/8</span>
+              </li>
+              <li>
+                Color 2 <span class="callout">/atem/preview/9</span>
+              </li>
+              <li>
+                Media Player 1 <span class="callout">/atem/preview/10</span>
+              </li>
+              <li>
+                Media Player 2 <span class="callout">/atem/preview/12</span>
+              </li>
+            </ul>
+            <p>
+              Feedback: Enabled for all values.
+            </p>
+            <h3>Transitions</h3>
+            <ul>
+              <li>
+                T-bar <span class="callout">/atem/transition/bar <0-1></span>
+              </li>
+              <li>
+                Cut <span class="callout">/atem/transition/cut</span>
+              </li>
+              <li>
+                Auto <span class="callout">/atem/transition/auto</span>
+              </li>
+              <li>
+                Fade-to-black <span class="callout">/atem/transition/ftb</span>
+              </li>
+            </ul>
+            <h3>Setting transition type</h3>
+            <ul>
+              <li>
+                Mix <span class="callout">/atem/transition/set-type/mix</span>
+              </li>
+              <li>
+                Dip <span class="callout">/atem/transition/set-type/dip</span>
+              </li>
+              <li>
+                Wipe <span class="callout">/atem/transition/set-type/wipe</span>
+              </li>
+              <li>
+                Stinger <span class="callout">/atem/transition/set-type/sting</span>
+              </li>
+              <li>
+                DVE <span class="callout">/atem/transition/set-type/dve</span>
+              </li>
+            </ul>
+            <p>
+              Feedback: None.
+            </p>
+            <h3>Auxiliary source selection</h3>
+            <ul>
+              <li>
+                Aux <span class="callout">/atem/aux/$i $x</span>
+                <ul>
+                  <li>Where <span class="callout">$x</span> is an integer value that is a valid program source, and can be 1-6 depending on the capability of your ATEM switcher. Check the Help Menu for the correct values.</li>
```

Comment:
This is probably an issue in the actual codebase as well, but it should be:

`<li>Where <span class="callout">$x</span> is an integer value that is a valid AUX output, and can be 1-6 depending on the capability of your ATEM switcher. Check the Help Menu for the correct values.</li>`